tests/foreman/ui/test_hostcollection.py

```diff
@@ -460,6 +461,7 @@ def setUp(self):
         self.hosts = []
         for _ in range(self.hosts_number):
             client = VirtualMachine(distro=DISTRO_RHEL7)
+            self.addCleanup(vm_cleanup, client)
```

Comment:
just a question. Is it gonna work in a loop? Like create a listener per each vm?